zkevm-circuits/Cargo.toml

```diff
@@ -47,4 +47,4 @@ rand_chacha = "0.3"
 
 [features]
 default = []
-test = []
+test = ["dep:mock"]
```

Comment:
I want to mention that I've never seen this, so if someone can provide documentation on the usage of `dep:` prefix, I would really appreciate it!

Reply:
Found no docs for this. I just removed the prefix :)